src/lib.rs

```diff
@@ -8,7 +8,6 @@
 //! * This implementation does not require the Rust standard library.
 //! * All operations are constant time unless explicitly noted.
 
-#![no_std]
 #![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
```suggestion
// This implementation now uses the Rust standard library because `Fp` derives `Hash`
// #![no_std]
#![cfg_attr(docsrs, feature(doc_cfg))]
```